File: FMAplication/Domain/Brand/SubBrand.cs
using FMAplication.Core;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FMAplication.Domain.Brand
{
    public class SubBrand : AuditableEntity<int>
    {
        public string Name { get; set; }
        public string Code { get; set; }
        public string Details { get; set; }

        public int BrandId { get; set; }
        public Brand Brand { get; set; }
    }
}

File: FMAplication/ClientApp/src/app/Shared/Entity/Brands/sub-brand.ts
import { Status } from "src/app/Shared/Enums/status";
import { Brand } from './brand';

export class SubBrand {
    public id: number;
    public brandId: number;
    public name: string;
    public code: string;
    public details: string;
    public status: number;
    public brand: Brand;

    constructor() {
        this.id = 0;
        // this.brandId = 0;
        this.name = '';
        this.code = '';
        this.details = '';
        this.status = Status.Active;
        // this.brand = new Brand();
    }
}